front/src/Api/IframeListener.ts

```diff
@@ -22,9 +21,21 @@ class IframeListener {
     private readonly _openPopupStream: Subject<OpenPopupEvent> = new Subject();
     public readonly openPopupStream = this._openPopupStream.asObservable();
 
+    private readonly _disablePlayerControlStream: Subject<void> = new Subject();
+    public readonly disablePlayerControlStream = this._disablePlayerControlStream.asObservable();
+
+    private readonly _enablePlayerControl: Subject<void> = new Subject();
+    public readonly enablePlayerControl = this._enablePlayerControl.asObservable();
```

Comment:
We should add the "stream" suffix to be coherent (and also in displayBubble and removeBubble)